packages/cli/jest-puppeteer.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  launch: {
+    headless: true, // set to false to view browser window
+  },
+  server: {
+    command: 'node index.js serve test/e2e/test_site -n -p 8888',
```

Comment:
missing `-d` (to test the latest changes of the PR)

Reply:
I discovered that this is not possible at the moment and reported the bug in #1891. Nonetheless, I think it is still valuable to test the current serve command without the `-d` (good to test it as well, perhaps later on), since the test cases that I have currently written should not be impacted by latest changes

Reply:
As a temporary workaround, try (in this example)

```
cd <test/e2e/test_site> && ... run the command
```